src/commands.rs

```diff
@@ -2913,11 +2938,20 @@ don't make any changes, then the operation will be aborted.
     } else {
         let mut tx =
             workspace_command.start_transaction(&format!("split commit {}", commit.id().hex()));
+        let middle_tree = workspace_command
+            .repo()
+            .store()
+            .get_tree(&RepoPath::root(), &tree_id)?;
+
         let first_description = edit_description(
             ui,
             tx.base_repo(),
-            &("JJ: Enter commit description for the first part.\n".to_string()
-                + commit.description()),
+            &_description_template_for_cmd_split(
+                &workspace_command,
+                "Enter commit description for the first part (parent).",
+                commit.description(),
+                base_tree.diff(&middle_tree, &EverythingMatcher),
+            )?,
```

Comment:
This expression is long enough that I think it would be clearer if it's extracted to a variable (same below, of course).